app/code/Magento/Indexer/Block/Backend/Grid/Column/Renderer/ScheduleStatus.php

```diff
@@ -0,0 +1,83 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\Indexer\Block\Backend\Grid\Column\Renderer;
+
+use Magento\Backend\Block\Widget\Grid\Column\Renderer\AbstractRenderer;
+use Magento\Framework\Mview\View;
+use Magento\Framework\Phrase;
+
+class ScheduleStatus extends AbstractRenderer
+{
+    /**
+     * @var \Magento\Framework\Mview\ViewInterface
+     */
+    protected $viewModel;
+
+    public function __construct(
```

Comment:
@fredden , can you please add the annotation block here? Thanks

Reply:
I've coped & modified the annotation block from the ancestor I'm overwriting, but the format doesn't match the other annotation block below. Please let me know if you want more/different content here.

Reply:
Look ok for me. Thanks